Add instructions to connect to Mailhog server in development docs. (#46)
fleetdm/fleet: docs/development/faq.md

```

If you get an `undefined: Asset` error it is likely because you did not run `make generate` before `make build`. See [Building the Code](https://github.com/fleetdm/fleet/blob/master/docs/development/building-the-code.md) for additional documentation on compiling the `fleet` binary.

## How do I connect to the Mailhog simulated mailserver?

> First, in the Fleet UI, navigate to the App Settings page under Admin. Then, in the "SMTP Options" section, enter any email address in the "Sender Address" field, set the "SMPT Server" to 'localhost' on port '1025,' and set "Authentication Type" to 'None.' Note that you may use any active or inactive sender address.

Reviewer: ```suggestion
First, in the Fleet UI, navigate to the App Settings page under Admin. Then, in the "SMTP Options" section, enter any email address in the "Sender Address" field, set the "SMTP Server" to `localhost` on port `1025`, and set "Authentication Type" to `None`. Note that you may use any active or inactive sender address.
```

> Visit `locahost:8025` to view Mailhog's admin interface which will display all emails sent using the simulated mailserver.

Reviewer: ```suggestion
Visit [locahost:8025](http://locahost:8025) to view Mailhog's admin interface which will display all emails sent using the simulated mailserver.
```

> Visit `locahost:8025` to view Mailhog's admin interface which will display all emails sent using the simulated mail server.

Reviewer: ```suggestion
Visit [locahost:8025](http://localhost:8025) to view Mailhog's admin interface which will display all emails sent using the simulated mail server.
```
